master/docs/developer/cls-forcesched.rst

```diff
@@ -1,128 +0,0 @@
-.. -*- rst -*-
-.. _ForceScheduler:
-
-ForceScheduler
---------------
-
-The force scheduler has a symbiotic relationship with the web status, so it deserves some further description.
-
-Parameters
```

Comment:
This one is not obsolete. it might be updated a ittle bit.
A detailed, developer orienter documentation for this module is for me still necessary